assets/js/mlmodels.js

```diff
@@ -0,0 +1,98 @@
+const { TypesenseInstantSearchAdapter, instantsearch } = window;
```

Comment:
The way these files get included by the browser, every variable in this file will cause a name collision. Try to change the variable names and remove this first line. They'll alsready be included by the models.js file. We'll want to make sure that these can't load in the wrong order, so we'll probably have to put the contents of this file in the models.js file after all, but you can keep it here for now to keep things separate.